src/main/java/it/pagopa/transactions/commands/data/AuthorizationRequestData.java

```diff
@@ -11,5 +13,9 @@ public record AuthorizationRequestData(
         String brokerName,
         String pspChannelCode,
         String paymentMethodName,
-        String pspBusinessName
+        String pspBusinessName,
```

Comment:
Could we improve this implementation using dtos inherited from RequestAuthorizationRequestDetailsDto interface